cmake/AddFdbTest.cmake

```diff
@@ -59,6 +62,9 @@ function(add_fdb_test)
   set(options UNIT IGNORE)
   set(oneValueArgs TEST_NAME TIMEOUT)
   set(multiValueArgs TEST_FILES)
+  if (NOT ENABLE_SIMULATION_TESTS)
```

Comment:
I think we should still include the simulation tests in the ensemble by default, but I think this change does not include the simulation tests by default.

Reply:
oh good, catch. I'll fix that

Reply:
fixed and verified manually that the files are in the correctness package